charts/cloudnative-pg/Chart.yaml

```
@@ -18,7 +18,7 @@ name: cloudnative-pg
 description: CloudNativePG Helm Chart
 icon: https://raw.githubusercontent.com/cloudnative-pg/artwork/main/cloudnativepg-logo.svg
 type: application
-version: "0.16.0"
+version: "0.16.1"
```

The release procedure will take care of bumping the version:
```suggestion
version: "0.16.0"
```